core/src/data_sources/splitter.rs

```diff
@@ -1005,4 +1016,24 @@ mod tests {
             .await
             .unwrap();
     }
+
+    // #[tokio::test]
```

Comment:
assuming we want to leave it here without running, i'm told we can use `#[ignore]` to skip the test, rather than comment it?

Reply:
I'll try to ignore it